### PR TITLE
Fix the issue where multiple bundlers get installed in toolchain

### PR DESCRIPTION
**Pull request by jaymalasinha**
## Description
We are using the omnibus-software (jsinha/remove_dependencies) branch to release a toolchain for all our platforms and also onboard Ubuntu-18.04 aarch64.  With the branch in omnibus-software we remove rubygems and bundler dependencies as it is already shipped with ruby 2.6 and above. We need exactly two versions of bundler in omnibus-toolchain to function with `bundle version` functionality so we install the latest bundler version using a custom bundler.rb for this repo

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

**Comment by jaymalasinha** [comment]
https://buildkite.com/chef/chef-omnibus-toolchain-master-omnibus-adhoc/builds/172. is the test build in progress and we use Jeremiah's branch https://github.com/chef/omnibus-buildkite-plugin/compare/snappj/skip-angry-install-for-ubuntu-aarch64.

File: .expeditor/release.omnibus.yml [hunk 1]
```diff
@@ -45,10 +45,11 @@ builder-to-testers-map:
   ubuntu-16.04-x86_64:
     - ubuntu-16.04-x86_64
     - ubuntu-18.04-x86_64
+  ubuntu-18.04-aarch64:
+    - ubuntu-18.04-aarch64
   windows-2012r2-i386:
     - windows-2012r2-i386
   windows-2012r2-x86_64:
-    - windows-2008r2-x86_64
```
Comment (tas50) [inline .expeditor/release.omnibus.yml]: Pour one out for win 2k8r2 🍺

Comment (jaymalasinha) [reply .expeditor/release.omnibus.yml]: 🍺

**Comment by jaymalasinha** [comment]
Closing this in favor of #134 .  Will cleanup rubygems and bundler as phase II of this effort.  Right now just getting minimal versions bundler 1.x and 2.x to unblock onboarding Ubuntu-18.04 aarch64

**Pull request closed without merging**